Automattic/jetpack
# Amazon block: Add an edit function

This adds a basic edit function to the Amazon block. There is more work to do on this, in future PRs. Until we have more idea of how the Amazon API works, we are using a dummy data file. I think it would be good to review/merge this before the PR gets any bigger, and continue in a new PR

#### Changes proposed in this Pull Request:

* Search for and add products from Amazon
* Change what fields are displayed
* Change color settings for the block

#### Screen capture
![Jan-30-2020 11-30-41](https://user-images.githubusercontent.com/275961/73446105-1bbe6d80-4354-11ea-9487-f9086b953707.gif)


#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* New feature, see pb5gDS-h1-p2

#### Testing instructions:






* Add an Amazon block to a post
* Check that you can search for things containing the word "face"
* Check that you can add a single product to the block
* Check that you can show/hide fields in the sidebar
* Check that you can change the color settings for a block
* Check that the block saves and is restored when you reload the page

#### Proposed changelog entry for your changes:

* no changelog
